Suggested workaround to build with recent gcc versions

Upstream bugzilla: https://bugs.freedesktop.org/show_bug.cgi?id=47792

Signed-off-by: Markos Chandras <markos.chandras@imgtec.com>

Index: xapp_luit-1.1.1/sys.c
===================================================================
--- xapp_luit-1.1.1.orig/sys.c
+++ xapp_luit-1.1.1/sys.c
@@ -20,6 +20,8 @@ OUT OF OR IN CONNECTION WITH THE SOFTWAR
 THE SOFTWARE.
 */
 
+#define _XOPEN_SOURCE 600
+
 #ifdef HAVE_CONFIG_H
 # include "config.h"
 #endif
